vm-init.d/qubes-updates-proxy

```diff
@@ -42,17 +42,16 @@ start() {
     have_qubesdb || return
 
     if qsvc qubes-updates-proxy ; then
-        # Yum proxy disabled
+        # Updates proxy disabled
         exit 0
     fi
 
     [ -x "$exec" ] || exit 5
     [ -f $config ] || exit 6
     # setup network redirection
     /sbin/iptables -I INPUT -i vif+ -p tcp --dport 8082 -j ACCEPT
```

Comment:
I think this line (and the matching one on stop) is not necessary either. The proxy is accessed over 'lo', not 'vif' interface.